dom/nodes/insertion-effects/Node-appendChild-script-and-style-from-fragment.tentative.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Node.appendChild: inserting a script and a style from a fragment</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id="log"></div>
+<script>
+const happened = []
+const style = document.createElement("style");
+let styleSheet = null;
+test(() => {
+  style.appendChild(new Text("body {}"));
+  const script = document.createElement("script");
+  script.textContent = `
+    happened.push(style.sheet === null);
```

Comment:
Maybe just insert `style.sheet` so that the value `null` gets added to `happened`? That's more clear than having to remember what "true" means when reading the expectations below.